skipchain/struct.go

```diff
@@ -511,11 +511,16 @@ func (fl *ForwardLink) Copy() *ForwardLink {
 // Verify checks the signature against a list of public keys. This list must
 // be in the same order as the Roster that signed the message.
 // It returns nil if the signature is correct, or an error if not.
-func (fl *ForwardLink) Verify(suite network.Suite, pubs []kyber.Point) error {
+func (fl *ForwardLink) Verify(suite cosi.Suite, pubs []kyber.Point) error {
 	if bytes.Compare(fl.Signature.Msg, fl.Hash()) != 0 {
 		return errors.New("wrong hash of forward link")
 	}
-	return fl.Signature.Verify(suite, pubs)
+	// this calculation must match the one in omnicon/bftcosi
```

Comment:
Then maybe they should be one and the same code... (yeah, ignore my last comment, sometimes I do not like copy/paste)